src/main/java/uk/gov/hmcts/reform/blobrouter/model/out/SearchResult.java

```diff
@@ -6,10 +6,14 @@
 
 public class SearchResult {
 
+    @JsonProperty("count")
+    public final int count;
+
     @JsonProperty("data")
     public final List<?> data;
 
     public SearchResult(List<?> data) {
         this.data = data;
+        count = (data == null) ? 0 : data.size();
```

Comment:
can use `CollectionUtils.size` which will handle even null :) but up to you

Reply:
which jar is it? I think it is not there.

Reply:
ok then let's leave it as it is

Reply:
what buggers me more is that List<?> data can be null flushed

p.s. i accidentally pressed `unresolve` button

Reply:
normally `data` can not be null

Reply:
so do we then need to check for nulls? it's not normal as you pointed out

Reply:
I prefer to check.